### PR TITLE
Fix code-samples after #310 release

### PR DESCRIPTION
**Pull request by brunoocasali**
Include a number of fixes after the latest breaking changes.

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -19,34 +19,34 @@ get_documents_1: |-
...
-      SearchQuery(
+      IndexSearchQuery(
         query: "us",
         indexUid: "movies_ratings"
       ),
     ]));
 faceted_search_2: |-
   await client.multiSearch(MultiSearchQuery(queries: [
-      SearchQuery(
+      IndexSearchQuery(
```
Comment (ahmednfwela) [inline .code-samples.meilisearch.yaml:39]: I think it's missing the query parameter

Comment (brunoocasali) [reply .code-samples.meilisearch.yaml:39]: Yep, I've opened an issue about it #321

Comment (brunoocasali) [reply .code-samples.meilisearch.yaml:39]: I've added the query params, but I think we should fix the issue in any case.

**Review by alallema — APPROVED** [review]
LGTM! 🌮

**Comment by brunoocasali** [comment]
bors merge

**Comment by brunoocasali** [comment]
bors cancel

**Pull request merged**